R/df_to_array.R

```diff
@@ -23,19 +23,6 @@ df.to.array <- function(
 #' @param value_var_name a [character()] variable containing a variable name from `df` which contains the values for the intended array.
 #' @return an [array()] with dimensions defined by the variables in `df` listed in `dim_var_names`
 #'
-#' @examples
-#' library(dplyr)
-#' library(tidyr)
-#'
-#' df <- iris %>%
-#'   tidyr::pivot_longer(
-#'     names_to = "parameter",
-#'     cols = c("Sepal.Length", "Sepal.Width", "Petal.Width", "Petal.Length")
-#'   ) %>%
-#'   mutate(parameter = factor(parameter, levels = unique(parameter)))
-#' arr <- df %>% serocalculator:::df.to.array(dim_var_names = c("parameter", "Species"))
-#' ftable(arr[,,1:5])
-#' @noRd
```

Comment:
Removed example from non-exported function, per CRAN request

Reply:
please undo, see:
- https://style.tidyverse.org/documentation.html#internal-functions, https://blog.r-hub.io/2019/12/12/internal-functions/, 
- https://r-pkgs.org/man.html#:~:text=documentation%2C%20include%20the-,%40noRd%20tag,-to%20suppress%20the,   
- https://devguide.ropensci.org/pkg_building.html#:~:text=Add%20%23%27-,%40noRd,-to%20internal%20functions 
 
for more